### PR TITLE
News and blog templates

### PR DESCRIPTION
**Pull request by davidrapson**

### CODE REVIEW

File: controllers/updates/views/post/blogpost.njk [hunk 1]
```diff
@@ -3,71 +3,73 @@
...
-                <p class="u-margin-top-s"><a href="{{ localify('/news/blog?author=' + author.slug) }}">See all posts by this author</a></p>
+                </div>
+            </aside>
+            <p class="u-margin-top-s"><a href="{{ localify('/news/blog?author=' + author.slug) }}">See all posts by this author</a></p>
+        {% endfor %}
+    {% endcall %}
+
+    {% call card("About this blog post", pageAccent) %}
+
+        <p class="article-meta__pubdate u-margin-bottom-s">
```
Comment (davidrapson) [inline controllers/updates/views/post/blogpost.njk]: Can this `article-meta__pubdate` class go too? It's a bit of a loose class and afaict only adds margin which the `u-` class overrides.

File: assets/sass/components/_news.scss [hunk 2]
```diff
@@ -166,3 +141,32 @@
     font-size: 18px;
     margin-bottom: 0;
 }
+
+/* =========================================================================
+   Blogpost Attribution 
+   ========================================================================= */
+
+.blogpost-attribution {
```
Comment (davidrapson) [inline assets/sass/components/_news.scss]: 💯

File: controllers/updates/views/post/blogpost.njk (hunk 1 above)
Comment (mattandrews) [reply controllers/updates/views/post/blogpost.njk]: Done!

**Comment by davidrapson** [comment]
I think we can get this merged now?

**Pull request merged**